integration-cli/docker_cli_ps_test.go

```diff
@@ -226,6 +226,48 @@ func (s *DockerSuite) TestPsListContainersFilterStatus(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestPsListContainersFilterHealth(c *check.C) {
+	// Test legacy no health check
+	out, _ := dockerCmd(c, "run", "--name=none_legacy", "-itd", "busybox")
```

Comment:
^^ :+1: The Windows busybox image can't be used interactively due to various reasons. So that container would just exit immediately. It sounds regardless, the wait shouldn't wait indefinitely and should have a reasonable timeout (1 min or so) for test failure.